### DISCUSSION TITLE
Get started with EJBCA Docker Container

### DISCUSSION THREAD

**Discussion by KarolinHem** [Keyfactor/ejbca-ce#164] [Announcements]
If you want to try out EJBCA as a Docker Container, then check out these guides with video tutorials that make it easy for you to get started: 
* [Quick Start Guide - Start EJBCA Container with Unauthenticated Network Access](https://doc.primekey.com/x/mpuKAw) 
* [Quick Start Guide - Issue Client Authentication Certificate using EJBCA](https://doc.primekey.com/x/sZuKAw) 

And feel free to comment below if you have any comments or questions.

**Comment by t984447** [comment]
Hejsan,

Thank you, Will use these as I got more time to futher setup EJBCA at home!
